lib/sqitch-check.pod

```diff
@@ -0,0 +1,113 @@
+=head1 Name
+
+sqitch-check - Checks for divergences between planned and deployed changes.
+
+=head1 Synopsis
+
+  sqitch check [options] [<database>]
+
+=head1 Description
+
+Runs various checks on the state of the working directory and the database.
```

Comment:
Is it "various checks" at this point? Seems like there's just the one at the moment; perhaps mention that more may be added in the future?